Source/Fuse.Reactive.Bindings/Instance.uno

```diff
@@ -145,10 +145,15 @@ namespace Fuse.Reactive
 			The `TemplateSource` property, along with the templates in the source, as well as the `TemplateKey`, must be set prior to
 			rooting to take effect.
 		*/
-		public Visual TemplateSource
+		public ITemplateSource TemplateSource
 		{
-			get; set;
+			get { return _weakTemplateSource; }
+			set { _weakTemplateSource = value; }
 		}
+		//https://github.com/fusetools/fuselibs-public/issues/135
+		[WeakReference]
+		ITemplateSource _weakTemplateSource;
```

Comment:
Is the comment about the `[WeakReference]` avoiding the memory-reference-loop in that issue? If so, maybe you could spell out what you mean?

Something like:
```uno
/*
  To avoid memory-loops between the template-source and the
  instantiatior, we keep a weak reference here until rooting
  is performed, and only keep a strong reference until we're
  unrooted again.
*/
```

I'm not a huge fan of pointing the code to the issue-tracker (generally I think information should flow in the other direction), as it makes the code harder to read if you're offline. There's obviously cases where we can't avoid it (tests disabled due to issues for instance), but I don't think this one of them.

This is just my personal preference, though. Take it or leave it ;)